### ISSUE TITLE
ConstantLine (xline yline) unsupported

### ISSUE THREAD

**Issue by LeoRauschenberger** [matlab2tikz/matlab2tikz#1066]
ConstantLine was introduced in 2018b:
https://de.mathworks.com/help/matlab/ref/matlab.graphics.chart.decoration.constantline-properties.html

when using for example:
`xline(4)`
This error appears:
`Error using matlab2tikz>handleAllChildren (line 730)
I don't know how to handle this object: constantline
Error in matlab2tikz>drawAxes (line 880)
    [m2t, childrenEnvs] = handleAllChildren(m2t, handle);
Error in matlab2tikz>saveToFile (line 482)
        m2t = drawAxes(m2t, relevantAxesHandle);
Error in matlab2tikz (line 334)
    m2t = saveToFile(m2t, fid, fileWasOpen);`

**Comment by egeerardyn** [comment]
Thank you for reporting this. I think this issue shows that we should have a different way of treating unknown objects: warn and ignore instead of throwing an error. That way, you can at least  convert the rest of the figure.

That being said, I think implementing a handler for that kind of object should be pretty straightforward. The only thing I'm unsure about is whether pgfplots has a native counterpart to this, but even a hacky attempt at this should be quite okay.

**Comment by evensure** [comment]
I just ran into the same problem while trying to get some tikz-plots for my thesis. Unfortunately I was using xline a lot already. 

In a german forum I found an approach for pgfplots using relative axes (answer 1) and relative positioning (answer 2) for xline and yline:
[https://texwelt.de/fragen/3317/wie-kann-ich-in-pgfplots-horizontale-oder-vertikale-hilfslinien-einzeichnen](url)

Unfortunately, to me implementing a handler is not straightforward, but at least now I have a way to manually edit the tikz-files after creation.
